frontend/src/features/landing_page/components/GeoMap.tsx

```diff
@@ -47,4 +50,29 @@ export const GeoMap = () => {
       {markers}
     </MapContainer>
   );
+
+  function handleClick() {
+    setIsVisible(true);
+  }
```

Comment:
I would prefer to inline this
```tsx
onClick={() => setIsVisible(true)}
```
or putting it to the top, above all the markup.

I think grouping the code like that makes it easier to read.